<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="dharani.TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="StandAloneTest(failed)">
    <classes>
      <class name="dharani.test.StandAloneTest">
        <methods>
          <include name="Close_Browser"/>
          <include name="SubmitOrder" invocation-numbers="1"/>
          <include name="LaunchApplication"/>
          <include name="OrderHistoryTest"/>
        </methods>
      </class> <!-- dharani.test.StandAloneTest -->
    </classes>
  </test> <!-- StandAloneTest(failed) -->
  <test thread-count="5" parallel="tests" name="ErrorValidation(failed)">
    <classes>
      <class name="dharani.test.ErrorValidationsTest">
        <methods>
          <include name="Close_Browser"/>
          <include name="LoginErrorValidation"/>
          <include name="LaunchApplication"/>
        </methods>
      </class> <!-- dharani.test.ErrorValidationsTest -->
    </classes>
  </test> <!-- ErrorValidation(failed) -->
</suite> <!-- Failed suite [Suite] -->
